hack/release.sh

```diff
@@ -397,4 +397,15 @@ main
 echo
 echo
 echo "Release complete; see $(s3_url)"
+echo "Use the following text to announce the release:"
+echo
+echo "We have just pushed $VERSION to test.docker.com. You can download it with the following:"
+echo
+echo "Ubuntu/Debian: curl -sSL https://test.docker.com/ubuntu | sh or curl -sSL https://test.docker.com/ | sh"
```

Comment:
lets not even mention the /ubuntu one

On Thu, Jul 9, 2015 at 11:10 AM, Sebastiaan van Stijn <
notifications@github.com> wrote:

> In hack/release.sh
> <https://github.com/docker/docker/pull/14502#discussion_r34284913>:
>
> > @@ -397,4 +397,15 @@ main
> >  echo
> >  echo
> >  echo "Release complete; see $(s3_url)"
> > +echo "Use the following text to announce the release:"
> > +echo
> > +echo "We have just pushed $VERSION to test.docker.com. You can download it with the following:"
> > +echo
> > +echo "Ubuntu/Debian: curl -sSL https://test.docker.com/ubuntu | sh or curl -sSL https://test.docker.com/ | sh"
>
> Should the /ubuntu minimum install be on a separate line? We currently
> have some open issues from people running the minimum install, ending up
> without AUFS.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/14502/files#r34284913>.
>